src/pages/sessions/running.js

```diff
@@ -229,55 +230,85 @@ const Running = ({ intl, shortname }) => (
+                                variables: {
+                                  sessionId: id,
+                                  settings: {
+                                    isFeedbackChannelActive: !settings.isFeedbackChannelActive,
+                                  },
                                 },
-                              }
-                            },
-                            variables: { sessionId: id },
-                          })
-                        }}
-                      />
+                              })
+                            }}
+                            handleDeleteFeedback={feedbackId => {
+                              deleteFeedback({
+                                variables: { feedbackId, sessionId: id },
+                              })
+                            }}
+                            handlePublicToggle={() => {
+                              updateSettings({
+                                refetchQueries: [{ query: RunningSessionQuery }],
+                                variables: {
+                                  sessionId: id,
+                                  settings: {
+                                    isFeedbackChannelPublic: !settings.isFeedbackChannelPublic,
+                                  },
+                                },
+                              })
+                            }}
+                            handleUpdateFeedback={(newTags, feedbackId) => {
+                              updateFeedback({
+                                refetchQueries: [{ query: RunningSessionQuery }],
+                                variables: {
+                                  feedbackId,
+                                  sessionId: id,
+                                  tags: newTags,
+                                  settings: {
+                                    isFeedbackChannelActive: settings.isFeedbackChannelActive,
+                                  },
+                                },
+                              })
+                            }}
+                            intl={intl}
+                            isActive={settings.isFeedbackChannelActive}
+                            isPublic={settings.isFeedbackChannelPublic}
+                            subscribeToMore={() => {
+                              subscribeToMore({
+                                document: FeedbackAddedSubscription,
+                                updateQuery: (prev, { subscriptionData }) => {
+                                  if (!subscriptionData.data) return prev
+                                  const newFeedbacks = [
+                                    ...prev.runningSession.feedbacks,
+                                    subscriptionData.data.feedbackAdded,
+                                  ]
+                                  const sortedFeedbacks = newFeedbacks.sort((f1, f2) => {
```

Comment:
Just use this sorting logic for all the feedbacks instead of also sorting in the API, as I mentioned in the PR there :)